src/views/ProfileInjectedTypeform.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Typeform from "../components/Typeform/typeform";
+import { useRecoilValue } from "recoil";
+import { auth_gsuite } from "../api/state";
+
+interface message {
+  typeform_id: string;
+}
+const ProfileInjectedTypeform = (props: message) => {
```

Comment:
Can we destructure the props here to avoid doing `const typeform_id = props.typeform_id;` later on. Something like this. Might need to experiment a little to get the syntax just right.

```
const ProfileInjectedTypeform = ({typeform_id} : message) => {
```